composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -48,111 +70,266 @@ fun MySpotsView(mySpotsViewModel: MySpotsViewModel, onAddSpotClick: () -> Unit)
+                                .clip(CircleShape)
+                                .size(64.dp)
                         ) {
-                            Text(
-                                text = "Edit Spot",
-                                color = Color.White
-                            )
+                            Text("+")
                         }
                     }
+                    Spacer(modifier = Modifier.height(16.dp))
+                }
+            }
+        }
+    }
+}
 
+//@Preview()
+@Composable
+fun AvailableDays() {
+    val weekDays = listOf("M", "T", "W", "T", "F", "S", "S")
+    var selectedWeekdays = mutableListOf(
+        true, true, true, true, true, false, false)
+    val itemShape = RoundedCornerShape(4.dp)
+    val orange = Color(234, 142, 112)
 
-                }
-                Spacer(modifier = Modifier.weight(1f))
+    Row (modifier = Modifier
+        .fillMaxWidth()
+        .background(Color.White),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        for (index in 0..6) {
+            val borderColor = if (selectedWeekdays[index]) orange else Color.Gray
+            val fontWeight = if (selectedWeekdays[index]) FontWeight.Bold else FontWeight.Thin
+            val bColor = if (selectedWeekdays[index]) Color.White else Color.LightGray
+
+            Text(
+                weekDays[index],
+                modifier = Modifier
+                    .weight(1f/7)
+                    .border(
+                        width = 1.dp,
+                        color = borderColor,
+                        shape = itemShape)
+                    .background(bColor, itemShape)
+                    .padding(vertical = 12.dp)
+                ,
+                textAlign = TextAlign.Center,
+                style = TextStyle(fontWeight = fontWeight)
+            )
+        }
+    }
+}
 
-                Button(
-                    onClick = onAddSpotClick,
-                    shape = CircleShape,
-                    colors = ButtonDefaults.buttonColors(backgroundColor = Color.LightGray),
+@Composable
+fun ImageCarousel() {
+    val scrollState = rememberScrollState()
+    val itemList = listOf("padding", "image1", "image2", "image3", "image4", "image6", "padding")
+    LazyRow(
+        horizontalArrangement = Arrangement.SpaceEvenly
+    ) {
+        items(itemList) { item ->
+            if (item == "padding") {
+                Spacer(modifier = Modifier.width(12.dp))
+            } else {
+                Box(
                     modifier = Modifier
-                        .align(Alignment.End)
-                        .clip(CircleShape)
-                        .size(64.dp)
-                ) {
-                    Text("+")
-                }
+                        .width(200.dp).height(154.dp)
+                        .padding(horizontal = 4.dp)
+                        .background(Color.LightGray),
+                )
+//                Image(
```

Comment:
this box with the images should be: 
               Box(
                    modifier = Modifier
                        .width(200.dp).height(154.dp)
                        .padding(horizontal = 4.dp)
                        .background(Color.LightGray),
                ) {
                    Image(
                        painter = painterResource(Res.drawable.image_placeholder),
                        contentDescription = null,
                        contentScale = ContentScale.FillBounds,
                    )
                }
This does stretch the image if they are not the same dimension ratio so you could address that if you'd like by removing the grey behind, but probably not necessary for now.

You'll need to import:
import cruiser.composeapp.generated.resources.image_placeholder